From a10a948228e47095a1be85bdbf66f445a5a8f26c Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Mon, 23 Mar 2020 17:00:53 -0500
Subject: [PATCH] kojira add sleeptime to conf

kojira no longer needs a patch to accept the sleeptime option, but it's
not mentioned in the default config file so fix that.
---
 util/kojira.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/util/kojira.conf b/util/kojira.conf
index 4d70695..2531b84 100644
--- a/util/kojira.conf
+++ b/util/kojira.conf
@@ -40,6 +40,9 @@ logfile=/var/log/kojira.log
 ;certificate of the CA that issued the HTTP server certificate
 ;serverca = /etc/kojira/serverca.crt
 
+;time to wait between repo checks (in seconds)
+;sleeptime = 15
+
 ;how soon (in seconds) to clean up expired repositories. 1 week default
 ;deleted_repo_lifetime = 604800
 
-- 
2.6.3

